crates/neon-runtime/Cargo.toml

```diff
@@ -5,15 +5,20 @@ authors = ["Dave Herman <david.herman@gmail.com>"]
 description = "Bindings to the Node.js native addon API, used by the Neon implementation."
 repository = "https://github.com/neon-bindings/neon"
 license = "MIT/Apache-2.0"
+edition = "2018"
 
 [dependencies]
 cfg-if = "0.1.9"
+libloading = { version = "0.6.5", optional = true }
 neon-sys = { version = "=0.5.3", path = "../neon-sys", optional = true }
-nodejs-sys = { version = "0.7.0", optional = true }
 smallvec = "1.4.2"
 
+[dev-dependencies]
+nodejs-sys = "0.7.0" # Dev dependency for easy copying
```

Comment:
Maybe be even more explicit in the comment, like "Not strictly needed; just here for easy manual copying"

Reply:
Done.